docs/docker-hub/index.md

```diff
@@ -16,23 +16,23 @@ management, user and team collaboration, and lifecycle workflow automation.
 
 ![DockerHub](/docker-hub/hub-images/hub.png)
 
-## [Finding and pulling images](./userguide.md)
+## [Finding and pulling images](userguide.md)
```

Comment:
I think we should remove the links from the headings. They also make the "toc" menu at the top right to contain deep-links instead of anchor-links.